cli/command/image/save.go

```diff
@@ -54,3 +61,14 @@ func runSave(dockerCli command.Cli, opts saveOptions) error {
 
 	return command.CopyToFile(opts.output, responseBody)
 }
+
+func validateOutputPath(path string) error {
+	dir := filepath.Dir(path)
+	if dir != "" && dir != "." {
+		_, err := os.Stat(dir)
+		if err != nil && os.IsNotExist(err) {
+			return errors.New("directory does not exist")
```

Comment:
Was there a reason for rewriting this error here? (not at my computer, but wondering if the original message contains the _path_ that was not found; which is hidden in this custom message)